pkgs/development/python-modules/versioneer/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi, ... }:
+
+
+buildPythonPackage rec {
+
+  name = "${pname}-${version}";
```

Comment:
no name

Reply:
No name? I think all(?) Python packages have names.. At least I've always given before and what I can see, other packages have name too..

Reply:
Or can you give me an example of a Python package that doesn't use `name` attribute?

Reply:
`name` is automatically filled in since a month or two

Reply:
Ok! :+1:

Reply:
Fixed.